OpenTabletDriver/Driver.cs

```diff
@@ -82,6 +82,11 @@ private set
         public DeviceReader<IDeviceReport> TabletReader { private set; get; }
         public DeviceReader<IDeviceReport> AuxReader { private set; get; }
 
+        public static IEnumerable<TabletConfiguration> GetPreCompiledConfigurations()
+        {
+            return CompiledTabletConfig.GetCompiledConfigs();
+        }
```

Comment:
Do this instead. It would provide a default method of grabbing tablet configurations which can be overridden to provide configurations manually, such as in the desktop implementation.

This also allows us to directly write a virtual `Detect()` method which will run through all configurations and match for each. Making this virtual allows the consumer to handle detection manually as well if desired.

```suggestion
        public virtual IEnumerable<TabletConfiguration> GetConfigurations() => CompiledTabletConfigurations.GetConfigurations();
```